### DISCUSSION TITLE
List Acordeon

### DISCUSSION THREAD

**Discussion by pihcardozo** [Telefonica/mistica-design#1327] [Proposals]
### Goal

Precisamos entender melhor se será um componente novo no Mística ou uma evolução do Row List existente. Se já existem estudos deste componente por parte da ES.

### Context

A solicitação vem da equipe de Produto, onde há a necessidade de ter filtros numa mesma página. Após vários estudos, chegamos a conclusão que um componente em formato de acordeon, supriria tais necessidades, ajudando a diminuir o scroll da página e manter a hierarquia das informações. 

### Impact

Atualmente utilizamos o row list para tal finalidade, mas sem a função de acordeon, adotando alternativas como utilizar o Botton Sheet como resposta ao toque/clique. Com um componente novo, teremos mais possibilidades de criação numa interface sem a necessidade de soluções alternativas. 

### Discussion Questions

Entendemos que a estrutura de um Acordeon seja muito similar ao Row List, seria interessante entender se seria um componente novo ou uma evolução de um Row List (Expand and collapse). Desta forma, também entender quais elementos poderíamos ter neste componente na área A, B e C, onde o icon-chevron down/up entraria como um elemento fixo.

### Additional considerations

Estudo feito no [Playroom](https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AhCAPABABwIZSgEsA7AcwF5gAmAFgF8A%2BAHWM01QximdbfbSx4CJCsACMANkYs%2BfJABUIAVzAALXACMANjBmy%2BEYgAUATjADO5ygAoAlJnINMwPfr7mYAFwDKn3J5hrJnB1Ml0QABpMAEJSL19-QOC1XDDg21sAblc3TA8fPwCgkNS4gElIYmCo2PjCpJKwisN0rJzZOjp2zB5c9jLiLRIYPLwwGHJgjS8AdxgYKpBevvZ5GHRPAGZMAFsuQiUdhnkAW88lLQgkAHo1jc3lvqRm4hQlT09DR5XgOILE4opJqVdKYAD8mCCvBWcheAGFVDAAG4mQwAVWwACUYKQLrgTAwbvDESj0VicXiCd03PZ4JDqblnpUEcjUcQACIQGbEbG4rT4wnXYmswyc7m8ynfPq2LrQxlCypvD5fBlEwbDKXsW7KFLaGDfG4Cb6-OoA5KhOKgiFQtwcQT4IhkSiSRhwwwBABfUAgmG9mAAoug8MQoJg3VotHgPIaMN9afS5WwkEQkYKU3HZUnrgIuDwY%2Bg8wkwABrQVF0ssSIgTyIvbmBAAbRAAFkIEjCOY-CYQABdCIgGaEKA1%2BvwBubCQABh7dCAA)

### Do you have a visual proposal?

https://www.figma.com/file/zBE7l1r0t19VHXsA7kQt92/List-Accordeon?type=design&node-id=110%3A14245&mode=design&t=MsuXA56ZDBhe654T-1

**Comment by yceballost** [comment]
Hola Fabio!

Este componente [ya está en el roadmap](https://github.com/Telefonica/mistica-design/issues/534) de Mística, es algo necesario a incluir en el sistema. 

Está algo avanzado pero ahora mismo está parado porque no tiene prioridad, la definición la empezó un diseñador del equipo de Smart Wifi. Invité a Design Ops Vivo a la branch para que podáis ver lo que hay y ver si lo que hay cubre vuestras necesidades.

https://www.figma.com/file/JT9bBc50ookse0Q0lCoKad/%23534---Accordion-component?type=design&node-id=0-1&mode=design

Cuando lo leas, puedes comentar por aquí si cubre vuestra necesidad (yo creo que sí) y así nos alineamos

**Comment by vinrosa** [comment]
Olá @yceballost acredito que o componente cobre todas as nossas necessidades para o app vivo, devemos abrir um ticket no JIRA para iniciarmos discutirmos o desenvolvimento?

**Reply by yceballost** [reply]
exacto, abrid el ticket para que pueda organizarse el desarrollo junto al resto del proyecto
